client/src/components/InputTextField/InputTextField.css

```diff
@@ -0,0 +1,41 @@
+.input-text-field-container input {
+  display: flex;
+  color: black;
+  width: 455px;
+  height: 55px;
```

Comment:
It'd be better to setting padding here to get a consistent sizing rather than to set an explicit height and width